chance/p5js/Popup-Circles/index.md

```diff
@@ -3,7 +3,7 @@ layout: p5js
 author: Daniele Maselli
 author-url: https://github.com/DanieleMaselli 
 title: Popup Circles 
-tags: [P5.js, chance, repeat, random, Law of Similarity]
+tags: [P5.js, chance, random, form, Law of Connected Elements]
```

Comment:
but is the problem that i allready used a law of similarty in my past pull requests?